### PR TITLE
Corpsman Glasses Adjustment

### PR DESCRIPTION
**Pull request by Avalon-Proto**
## About the PR
Removed Sechud From corpsman glasses

## Why / Balance
Mostly to provide the corpsman with flash protection, but also giving them a medhud. Should be more concerned with the healing than the hurting

## Technical details
Removed Sechud part of the corpsman glasses, made them craftable from just a medhud

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- tweak: Removed Sechud From Corpsman Glasses. Stick some secglasses in your pocket :blunt:

### CODE REVIEW

**Comment by Stop-Signs** [comment]
Woah, a PR that actually adjusts something instead of outright removing it and sending it to review hell, YIPPIE!!!!!!!!!!! all jokes aside nice PR avalon

File: Resources/Prototypes/DeltaV/Recipes/Construction/Graphs/clothing/glasses_corpshud.yml [hunk 1]
```diff
@@ -12,10 +12,10 @@
                 sprite: Clothing/Eyes/Glasses/sunglasses.rsi
                 state: icon
               doAfter: 5
-            - tag: HudMedicalSecurity
-              name: medsec hud
+            - tag: HudSecurity
```
Comment (TadJohnson00) [inline Resources/Prototypes/DeltaV/Recipes/Construction/Graphs/clothing/glasses_corpshud.yml]: This needs to be a medical hud, not a security hud

**Review by TadJohnson00 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Recipes/Construction/Graphs/clothing/glasses_corpshud.yml (hunk 1 above)
Comment (Avalon-Proto) [reply Resources/Prototypes/DeltaV/Recipes/Construction/Graphs/clothing/glasses_corpshud.yml]: Woops, on it

**Review by TadJohnson00 — CHANGES_REQUESTED** [review]
Last request: uncomment out the glasses from the corpsman lockers? Pretty please?

**Comment by Avalon-Proto** [comment]
> Last request: uncomment out the glasses from the corpsman lockers? Pretty please?

Forgot to say I did it woops

**Comment by TadJohnson00** [comment]
cleaned up comments and uncommented them out in the unfilled locker as well for ya.

**Review by TadJohnson00 — APPROVED** [review]

**Pull request merged**